### DISCUSSION TITLE
Add a Signal App bridge

### DISCUSSION THREAD

**Discussion by nanu-c** [chatwoot/chatwoot#8456] [Feature Requests]
**Is your enhancement request related to a problem? Please describe.**
I want to establish a trustworthy relationship with my customer, especially the ones that care for security and dislike other solutions

**Describe the solution you'd like**
Let's build a signal bridge with https://pkg.go.dev/github.com/signal-golang/textsecure and their reference implementation for a a cli, that could be extended to call the api.

**Describe alternatives you've considered**
Use a telegram-bot, but it's also not so secure.

**Comment by clanto007** [comment]
Can be used also this gateway to connect to signal that is the same used by Nextcloud:
https://gitlab.com/morph027/signal-cli-dbus-rest-api
